spec/sorbet_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+require 'open3'
+
+RSpec.describe 'sorbet' do
+  let!(:harry) do
+    Wizard.create!(
+      name: 'Harry Potter',
+      house: :Gryffindor,
+    )
+  end
+
+  before(:each) do
+    Rake::Task['rails_rbi:routes'].invoke
+    Rake::Task['rails_rbi:models'].invoke
+  end
+
+  it 'returns expected sorbet tc result' do
+    # invoke the rake tasks
+    cmd = "cd #{Rails.root.to_path}\; srb tc"
+    stdout, stderr, status = Open3.capture3(cmd)
```

Comment:
This should instead be something like `Open3.capture3('srb', 'tc')`. Entire commands should never be passed to `Open3` as a single string for security reasons. An additional argument of `:chdir => Rails.root.to_path` can handle the directory change as well.